.gitignore

```diff
@@ -85,11 +85,3 @@ sw.*
 
 # macOS
 .DS_Store
```

Comment:
Nit: We should probably also remove this `.DS_Store` because it should be part of every developers' global gitignore. (Otherwise, we have to add it to every repository we work in. Plus it's different per OS that a developer works in and not related to the actual project files.)

Reply:
Oh and also the `.nuxt`, `.vuepress/dist`,  and`.idea`?